test/studies/hpcc/HPL/vass/hpl.hpcc2012.notest

```diff
@@ -0,0 +1,8 @@
+This test fails to work with the new "deallocate arrays at the end of
```

Comment:
Isn't this one of the hpl versions that we report on and do multi-locale perf testing on